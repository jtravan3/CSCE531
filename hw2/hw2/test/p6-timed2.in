x=5
x
(#2+#2)%101
(#3+#3)%101
(#4+#4)%101
(#5+#5)%101
(#6+#6)%101
(#7+#7)%101
(#8+#8)%101
(#9+#9)%101
(#10+#10)%101
(#11+#11)%101
(#12+#12)%101
(#13+#13)%101
(#14+#14)%101
(#15+#15)%101
(#16+#16)%101
(#17+#17)%101
(#18+#18)%101
(#19+#19)%101
(#20+#20)%101
(#21+#21)%101
(#22+#22)%101
(#23+#23)%101
(#24+#24)%101
(#25+#25)%101
(#26+#26)%101
(#27+#27)%101
(#28+#28)%101
(#29+#29)%101
(#30+#30)%101
(#31+#31)%101
(#32+#32)%101
(#33+#33)%101
(#34+#34)%101
(#35+#35)%101
(#36+#36)%101
(#37+#37)%101
(#38+#38)%101
(#39+#39)%101
(#40+#40)%101
(#41+#41)%101
(#42+#42)%101
(#43+#43)%101
(#44+#44)%101
(#45+#45)%101
(#46+#46)%101
(#47+#47)%101
(#48+#48)%101
(#49+#49)%101
