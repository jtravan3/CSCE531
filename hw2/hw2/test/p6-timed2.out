Welcome to the Simple Expression Evaluator.
Enter one expression per line, end with ^D

   1: 5
   2: 5
   3: 10
   4: 20
   5: 40
   6: 80
   7: 59
   8: 17
   9: 34
  10: 68
  11: 35
  12: 70
  13: 39
  14: 78
  15: 55
  16: 9
  17: 18
  18: 36
  19: 72
  20: 43
  21: 86
  22: 71
  23: 41
  24: 82
  25: 63
  26: 25
  27: 50
  28: 100
  29: 99
  30: 97
  31: 93
  32: 85
  33: 69
  34: 37
  35: 74
  36: 47
  37: 94
  38: 87
  39: 73
  40: 45
  41: 90
  42: 79
  43: 57
  44: 13
  45: 26
  46: 52
  47: 3
  48: 6
  49: 12
  50: 24
  51: 
